api-doc.md

```diff
@@ -0,0 +1,301 @@
+            "place": "surat",
+            "references": "From canopas",
+            "message": "I'm a very good programer"
+        }
+```
+
+### 4. Send career mail
+
+```
+- Method : POST
+- Endpoint : /api/send-contact-mail
+- Description : API for send contact mail
+- Request:
+    - Headers : none
+    - Body :
+        {
+            "email": "shruti@gmail.com",
+            "name": "shruti",
+            "designation": "I am individual entrepreneur running my own business.",
+            "reason": "I have a rough design for the product I want to develop.",
+            "message": "just testing purpose",
+            "designation_info": "I am an owner of the business and I run canopas business. For more information about my business, you can visit the following links.",
+            "idea": "I have an idea for my business that I want to implement with your help.",
+            "social_media_links":
+                {
+                    "website": "https://canopas.com/",
+                    "facebook": "https://www.facebook.com/canopassoftware","twitter": "https://twitter.com/canopassoftware" ,"instagram": "https://www.instagram.com/canopassoftware/"
+                },
+            "contact_type": "Chat or Email"
+
+        }
+
+
+- Response :
+
+    - If API is not found or contact details data not found then,
+    - Status Code: 404 not found
+    - Data: no response data
+
+    - If request data(input) is not valid then,
+    - Status Code: 400 Bad Request
+    - Data: {"message" : "Invalid data"}
+
+    - If any server error is there then,
+    - Status Code: 500 Internal server error
+    - Data: no response data
+
+    - If request will success ,
+    - Status Code: 200 Ok
+    - Headers : none
+    - Data :
+        {
+            "name": "shruti",
+            "designation": "I am individual entrepreneur running my own business.",
+            "designation_info": "I am an owner of the business and I run canopas business. For more information about my business, you can visit the following links.",
+            "social_media_links": {
+                "facebook": "https://www.facebook.com/canopassoftware",
+                "instagram": "https://www.instagram.com/canopassoftware/",
+                "twitter": "https://twitter.com/canopassoftware",
+                "website": "https://canopas.com/"
+            },
+            "idea": "I have an idea for my business that I want to implement with your help.",
+            "reason": "I have a rough design for the product I want to develop.",
+            "email": "shruti@gmail.com",
+            "message": "just testing purpose",
+            "contact_type": "Chat or Email"
+}
+```
+
+### 5. Get sitemap
+
+```
+- Method : GET
+- Endpoint : /sitemap
+- Description : API for generate sitemap
+- Request:
+    - Headers : none
+    - Body :
+        {
+            "baseUrl": "http://canopas.com",
+        }
+
+
+- Response :
+
+    - If API is not found or sitemap data not found then,
```

Comment:
`If API is not found or sitemap data not found then,` => `If API is not found then,`